python/bufr/encoders/zarr/encoder.py

```diff
@@ -10,14 +10,16 @@
 
 
 # Encoder for Zarr format
-class Encoder:
-    def __init__(self, description: Union[str, bufr.encoders.Description]):
+class Encoder(bufr.encoders.EncoderBase):
+    def __init__(self, description:Union[str, bufr.encoders.Description]):
         if isinstance(description, str):
             self.description = bufr.encoders.Description(description)
         else:
             self.description = description
 
-    def encode(self, container: bufr.DataContainer, output_path: str) -> dict[tuple[str],zarr.Group]:
+        super(Encoder, self).__init__(self.description)
```

Comment:
I'm a little confused as to the purpose of this?

Reply:
`super` calls the base class implementation of the constructor, the base class keeps track of the description.